test/e2e/tests/metrics/state-snapshots/errors-after-init-opt-in-background-state.json

```diff
@@ -376,8 +386,8 @@
   },
   "UserOperationController": { "userOperations": "object" },
   "UserStorageController": {
-    "isProfileSyncingEnabled": null,
```

Comment:
This is interesting. The snapshot asserted that profile syncing was null, and now it is true. You might be right about the migrations.